.github/workflows/build_rpm.yaml

```diff
@@ -0,0 +1,137 @@
+name : Build RPM pipeline
+on:
+  workflow_call:
+    inputs:
+      target-architecture:
+        description: 'What is the architecture we want to build for'
+        required: true
+        type: string
+      build-version:
+        description: 'What is the version used to build and tag the RPMs'
+        required: true
+        type: string
+
+jobs:
+  parsing:
+    runs-on: ubuntu-latest
+    outputs:
+      name: ${{ steps.params.outputs.name }}
+    steps:
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-tags: 'true'
+      
+      - name: Setup for alumet
+        run: |
+          mkdir -p alumet
+
+      - name: Checkout alumet
+        uses: actions/checkout@v4
+        with:
+            repository: alumet-dev/alumet
+            path: "alumet"
+      
+      - name: Parse alumet version
+        id: params
+        run: |
+          name=$(awk '/Name/ { print $2; exit }' ./specs/alumet.spec)
+
+          echo "name=$name" >> $GITHUB_OUTPUT
+
+          echo "Pipeline will build [$name] in version [${{ inputs.build-version }}] for the architecture [${{ inputs.target-architecture }}]" >> $GITHUB_STEP_SUMMARY
+
+  packaging:
+    runs-on: ubuntu-latest
+    needs:
+      - parsing
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["fedora:40", "fedora:41", "fedora:42", "registry.access.redhat.com/ubi8/ubi:8.10", "registry.access.redhat.com/ubi9/ubi:9.5"]
+    env:
+      name: ${{ needs.parsing.outputs.name }}
+    container:
+      image: ${{ matrix.os }}
+    steps:
+      
+      - uses: actions/checkout@v4
+      
+      - name: Export good OS version
+        run: |
+          echo "OS=$(echo ${{ matrix.os }} | awk -F'[/:]' '{print $(NF-1) "." $NF}')" >> $GITHUB_ENV
+
+      - name: Install RPM on Fedora
+        if: contains(matrix.os, 'fedora')
+        run: |
+          sudo dnf install rust cargo -y
+          export PATH=$PATH:$HOME/.cargo/bin/
+          sudo dnf install rpm-build -y
+          sudo dnf install cargo-rpm-macros -y
+          sudo dnf install openssl-devel -y
+
+      - name: Install RPM on other than Fedora
+        if: ${{ ! contains(matrix.os, 'fedora') }}
+        run: |
+          packages=("unzip" "rpm-build" "openssl-devel" "curl" "rust" "cargo")
+          for package in "${packages[@]}"; do
+            if ! yum list installed "$package" >/dev/null 2>&1; then
+              yum install -y --allowerasing "$package"
+            fi
+          done
+
+        
+      - name: Install protoc
```

Comment:
Won't be needed once the new relay mode is merged.

Reply:
When will you merge it ?

Reply:
soon :)